scripts/simtest/cargo-simtest

```diff
@@ -76,7 +76,7 @@ if [ -n "$USE_MOCK_CRYPTO" ]; then
   )
 fi
 
-RUST_FLAGS=$(IFS=, ; echo "${RUST_FLAGS[*]}")
+rust_flags_str=$(IFS=, ; echo "${RUST_FLAGS[*]}")
```

Comment:
where the changes to this file intended for this PR as well?

Reply:
Yeah they were just warnings from my linter